projects/packages/my-jetpack/_inc/components/stats-section/test/create-stat-diff-text.js

```diff
@@ -0,0 +1,54 @@
+import { _n } from '@wordpress/i18n';
+import createStatDiffText from '../create-stat-diff-text';
+
+// Mock formatNumber and formatPercentage to make tests predictable
+jest.mock( '../../../utils/format-number', () => number => number.toString() );
+jest.mock( '../../../utils/format-percentage', () => ratio => `${ ratio * 100 }%` );
+
+describe( 'createStatDiffText', () => {
```

Comment:
Love the tests!
If you do add checks for non numeric counts / previous counts in `CreateStatDiffText` , it could also be worth adding a test to check that we handle them properly as well.